http/server.go

```diff
@@ -399,15 +353,101 @@ func (s *Server) handleExec(ctx *fasthttp.RequestCtx) {
 	s.replyOK(ctx)
 }
 
+func (s *Server) handleSimpleJsonQuery(ctx *fasthttp.RequestCtx) {
+	username, password := s.extractBasicAuth(ctx)
```

Comment:
See [here](https://github.com/v3io/frames/blob/development/http/server.go#L412), was added lately.
Also Go common practice is to call it `handleSimpleJSONQuery` (JSON)